rules/windows/collection_posh_audio_capture.toml

```diff
@@ -17,21 +17,29 @@ note = """## Triage and analysis.
 
 ### Investigating PowerShell Suspicious Script with Audio Capture Capabilities
 
-PowerShell is one of the main tools used by system administrators for automation, report routines, and other tasks.
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks, making
+it available for use in various environments, creating an attractive way for attackers to execute code.
```

Comment:
```suggestion
it available for use in various environments, and creates an attractive way for attackers to execute code.
```

Reply:
Adjusted in https://github.com/elastic/detection-rules/pull/1805/commits/52ecea470fe21a120e05d2878c42dfc10d5806ea